### ISSUE TITLE
Verbose "key was in the reservedKeys set, and was dropped"

### ISSUE THREAD

**Issue by josephyanks** [DataDog/dd-sdk-android#881]
Thanks for taking the time for reporting an issue!

**Describe what happened**
Include any error message or stack trace if available.

We are seeing a constant barrage of messages about timestamps being dropped because they are in the reservedKeys set.

```
2022-03-25 11:48:15.849 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:15.925 28970-29198/ E/Datadog: "_dd.timestamp=1648223153265" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:15.965 28970-29198/ E/Datadog: "_dd.timestamp=1648223203796" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.014 28970-29198/ E/Datadog: "_dd.timestamp=1648223254330" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.031 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.087 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.122 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.146 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.183 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.204 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.228 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
2022-03-25 11:48:16.251 28970-29198/ E/Datadog: "_dd.timestamp=1648222788501" key was in the reservedKeys set, and was dropped.
```

It looks like it's because the ignoredAttributes in the RumEventSerializer contains `RumAttributes.INTERNAL_TIMESTAMP`

The source of this INTERNAL_TIMESTAMP appears to be in `BridgeRum#startResource`, which is what our react native (@Datadog/mobile-react-native module) is calling `DdRum#startResource`, which in turn calls `BridgeRum#startResource`, where it creates this attributes map with the reservedKey:
```
        val attributes = context.toMutableMap().apply {
            put(RumAttributes.INTERNAL_TIMESTAMP, timestampMs)
        }
```

**Steps to reproduce the issue:**
I can't share the source code for our app, but let me know if more information than the above is needed.

**Describe what you expected:**

I'm not sure if this is a real issue or not but at the very least the logging should be removed for this case - we are getting a large number of these errors when using the RUM logging.

It seems like either the logging should be removed for the reserved `_dd.timestamp`, or the BridgeRum#startResource should use a key that's different from the reserved key for timestamp.

**Additional context**
 - Android OS version: API 31 (Android 12)
 - Device Model: Pixel 2
 - Datadog SDK version: 1.11.1
 - Versions of any other relevant dependencies (OkHttp, …):   `@datadog/mobile-react-native` : `"^1.0.0-rc4"`
 - Proguard configuration: N/A, this is in debug at the moment without proguard rules
 - Gradle Plugins: `com.datadoghq:dd-sdk-android-gradle-plugin:1.1.0`

**Comment by 0xnm** [comment]
Hello @josephyanks! This issue was fixed recently in #874 and the fix will be available in the next release of our React Native SDK.

**Comment by josephyanks** [comment]
That's great news, thank you.
